x/nft/username/bucket.go

```diff
@@ -16,7 +19,8 @@ type Bucket struct {
 
 func NewBucket() Bucket {
 	return Bucket{
-		Bucket: nft.WithOwnerIndex(orm.NewBucket(BucketName, NewUsernameToken(nil, nil, nil))),
+		Bucket: nft.WithOwnerIndex(orm.NewBucket(BucketName, NewUsernameToken(nil, nil, nil))).
+			WithMultiKeyIndex(ChainAddressIndexName, chainAddressIndexer, true),
```

Comment:
cool, global unique address lookup.
:+1:

Reply:
Is this tested somewhere? Probably it is, I just missed it, please link to lines so I can review